admin/attributes_controller.php

```diff
@@ -1569,7 +1569,7 @@ function popupWindow(url) {
                     ?>
                     <td class="text-right"><?php echo $attributes_price_final . $new_attributes_price . ' ' . $attributes_price_final_onetime; ?></td>
                     <?php if ($action != '') { ?>
-                      <td width="120">&nbsp;</td>
```

Comment:
Arbitrarily removing an attribute "just because it's 'deprecated'" doesn't necessarily mean it's the only thing needed.

Why was a width set on this column in the first place? How has that intention been accommodated in the "new" way that the "deprecation" intended? 
ie: if "width" is deprecated it usually means the intended new way is to use CSS or something to replace it. Where's the corresponding CSS to retain the design intention?

Reply:
To put this in context, this td is the last column displayed when Editing an attribute:

![td120](https://user-images.githubusercontent.com/4391026/100937458-530d2600-34f3-11eb-9e2a-c4167efb4736.gif)

The 120 width is irrelevant as the column width is determined by the "Action" heading text.

Here the 120 is removed, and there seems little visual difference.

![td120](https://user-images.githubusercontent.com/4391026/100937649-96679480-34f3-11eb-9465-c4deffdca328.gif)

So apart from deprecated, it seems unneeded,